### PR TITLE
CastleNathria/Trash: Add Sin Bolt Volley

### PR DESCRIPTION
**Pull request by psykzz**
This partially handles #1029.
I wanted to keep my first contribution small in case there was more feedback.

### CODE REVIEW

File: CastleNathria/Trash.lua [hunk 1]
```diff
@@ -32,6 +32,8 @@ mod:RegisterEnableMob(
 	173609, -- Nathrian Conservator
 	173633, -- Nathrian Archivist
```
Comment (Justw8) [inline CastleNathria/Trash.lua:34]: Don't need an emptyline here

File: CastleNathria/Trash.lua [hunk 2]
```diff
@@ -112,6 +115,7 @@ function mod:GetOptions()
 		{342770, "EMPHASIZE"}, -- Eradication Seeds
 		{339975, "TANK_HEALER"}, -- Grievous Strike
 		{342752, "HEALER"}, -- Weeping Burden
+		341146, -- Sin bolt volley
```
Comment (Justw8) [inline CastleNathria/Trash.lua:118]: Should be: "Sin Bolt Volley"

**Comment by Justw8** [comment]
Looks fine overall!

**Comment by funkydude** [comment]
Nice work!

**Pull request merged**